packages/sources/por-address-list/src/transport/bedrockUniBTC.ts

```diff
@@ -89,14 +89,32 @@ const getAddresses = (
           chainId: 'mainnet',
         }))
         .sort()
-    case 'tokens':
-      return Object.entries(data.evm)
-        .map(([_, value]) => ({
-          chainId: value.chain_id.toString(),
-          contractAddress: value.vault,
-          wallets: value.tokens.filter((t) => t != VAULT_PLACEHOLDER).sort(),
-        }))
-        .sort()
+    case 'tokens': {
+      const list = Object.entries(data.evm)
+        .flatMap(([_, value]) =>
+          value.tokens.map((token) => ({
+            chainId: value.chain_id.toString(),
+            contractAddress: token,
+            wallet: value.vault,
+          })),
+        )
+        .filter((item) => item.contractAddress != VAULT_PLACEHOLDER)
```

Comment:
why not just end it here with `wallets` instead of `wallet`?

Reply:
Since this is a reverse mapping, we are mapping
{
1, [2, 3]
}

into 
{
[2, 1]
[3, 1]
}

So need to flatten the whole thing first